spec/Filter/Filter.BinarySpatial.Implementations.js

```diff
@@ -0,0 +1,74 @@
+describe('Filter.BinarySpatial', function () {
```

Comment:
More complex tests with defined CRS (with straight and inverted axis order) and some real geometries are really needed, you just separately test that filters returns expected wrappers, but you don't test full filter use cases.